server/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -484,8 +493,13 @@ public boolean lenient() {
      * Set a cutoff value in [0..1] (or absolute number &gt;=1) representing the
      * maximum threshold of a terms document frequency to be considered a low
      * frequency term.
+     *
+     * @deprecated Since max_score optimization landed in 7.0, normal MultiMatchQuery
+     *             will achieve the same result without any configuration.
      */
+    @Deprecated
     public MultiMatchQueryBuilder cutoffFrequency(float cutoff) {
+        deprecationLogger.deprecated(CUTOFF_FREQUENCY_DEPRECATION_MSG);
```

Comment:
I don't think you need to use the deprecation logger here. The Deprecated annotation should be enough.